contrib/node/src/python/pants/contrib/node/tasks/node_task.py

```diff
@@ -77,40 +79,54 @@ def execute_node(self, args, workunit_name, workunit_labels=None, node_paths=Non
                                  workunit_name=workunit_name,
                                  workunit_labels=workunit_labels)
 
-  def execute_npm(self, args, workunit_name, workunit_labels=None, node_paths=None):
-    """Executes npm passing the given args.
+  def add_package(
+    self, target=None, package_manager=None, 
+    package=None, type_option=None, version_option=None,
+    node_paths=None, workunit_name=None, workunit_labels=None):
+    """Add an additional package using requested package_manager."""
```

Comment:
Would it be helpful for this to take a `**kwargs` argument to forward to either the `.add_package()` or `._execute_command()` calls below? I see that `node_paths` is unused and I feel like this many keyword args gets unwieldy. Named options shared between different methods is also something that could be encapsulated in a `pants.util.objects.datatype` subclass. Just wanted your thoughts on whether that would be a useful change, no action needed here.